packages/okta-angular/src/okta/services/okta.service.ts

```diff
@@ -63,14 +63,14 @@ export class OktaAuthService {
     /**
      * Returns the OktaAuth object to handle flows outside of this lib.
      */
-    getOktaAuth() {
+    getOktaAuth(): OktaAuth {
       return this.oktaAuth;
     }
 
     /**
      * Checks if there is a current accessToken in the TokenManager.
      */
-    isAuthenticated() {
+    isAuthenticated(): boolean {
       return !!this.oktaAuth.tokenManager.get('accessToken');
     }
 
```

Comment:
Not sure what the type definitions of `getAccessToken` and `getIdToken` should be. Does `@okta/okta-auth-js` contain a structure for it?

Reply:
Looks like #115 corrects it.